src/main-client.js

```diff
@@ -0,0 +1,120 @@
+import $ from 'jquery'
+import moment from 'moment'
+import Hotel from './Hotel';
+import Booking from './Booking';
+import Room from './Room';
+
+let url, urlString, userId, hotel
```

Comment:
Couple places where semicolons are missing, not breaking, but probably stylistically worth adding.